kong.conf.default

```diff
@@ -385,6 +385,19 @@
                                  # to be sent across from CP to DP in Hybrid mode
                                  # Default is 4Mb - 4 * 1024 * 1024 due to historical reasons
 
+#cluster_dp_labels =             # Labels for the data plane. These key-value entries are
+                                 # exposed via the clustering/data_planes endpoint.
+                                 # The value must be configured as a string in JSON format.
+                                 #
+                                 # Labels are only compatible with hybrid mode deployments on Konnect,
+                                 # they are not exposed on the clustering/data_planes endpoint.
+                                 #
+                                 # Keys and values follow the AIP standards:
+                                 # https://kong-aip.netlify.app/aip/129/
```

Comment:
```suggestion
#cluster_dp_labels =             # Labels for the data plane. These JSON-encoded key-value provide
                                 # additional context information for each DP.
                                 # The value must be configured as a string in JSON format.
                                 #
                                 # Labels are only compatible with hybrid mode
                                 # deployments with Kong Konnect (SaaS),
                                 # this configuration doesn't work with
                                 # self-hosted deployment.
                                 #
                                 # Keys and values follow the AIP standards:
                                 # https://kong-aip.netlify.app/aip/129/
```

Please fix spacing.

Reply:
Clarifying that only string keys are supported would be helpful.
I don't trust the linked web URL to remain static.

Reply:
fixed. I don't see an alternative for the URL, if you have any please let me know and I will replace it.